toolchain/sem_ir/name_scope.h

```diff
@@ -61,18 +63,23 @@ class NameScope : public Printable<NameScope> {
   }
 
   // Adds a new name known to not exist.
+  //
   // Must not be poisoned.
   auto AddRequired(Entry name_entry) -> void;
 
   // If the given name already exists, return true and an EntryId.
+  //
   // If the name is found, it might be a poisoned name.
+  //
   // If the name is not found, adds the name using inst_id and access_kind and
   // returns false and an EntryId.
+  //
+  // inst_id must not be poisoned.
```

Comment:
```suggestion
  // If the given name already exists, return true with the EntryId; the entry
  // might be poisoned. Otherwise, adds the name using inst_id and access_kind
  // and returns false with the new EntryId.
  //
  // This cannot be used to add poisoned entries; use AddPoison instead.
```

Suggesting a slightly different phrasing here, since I'm suggesting the paragraph edit too.

Reply:
Done.